mergify_cli/__init__.py

```diff
@@ -28,11 +28,11 @@
 import rich
 import rich.console
 
-from git_push_stack.commit_msg_hook import COMMIT_MSG_HOOK
+from mergify_cli.commit_msg_hook import COMMIT_MSG_HOOK
 
 
 try:
-    VERSION = importlib.metadata.version("git-push-stack")
+    VERSION = importlib.metadata.version("mrgfy")
```

Comment:
Shouldn't it be `mergify_cli` like the rest ?

```suggestion
    VERSION = importlib.metadata.version("mrgfy")
```